di_website/templates/includes/heroes/hero.html

```diff
@@ -1,7 +1,7 @@
 {% load wagtailimages_tags wagtailcore_tags responsive %}
 
 {% responsiveimage page.hero_image fill-1600x780-c100 srcset=global.hero_srcs as image %}
-<section class="hero{% if image %} hero--image{% endif %} {% block hero_class %}{% endblock %}">
+<section class="hero hero--image {% block hero_class %}{% endblock %}">
```

Comment:
Nope ... the `hero--image` class is appropriately conditional. This will break a lot of stuff that function recently complained about.

Reply:
@edwinmp I noticed that `hero--image` floats the author meta to the right according to pattern template design; though as i have just read through [#579](https://github.com/devinit/DIwebsite-redesign/issues/579) you are definitely right.